CoreMapView/src/au/gov/asd/tac/constellation/views/mapview2/layers/DayNightLayer.java

```diff
@@ -169,13 +170,15 @@ private String generatePath(final List<Location> locations) {
 
             // If is the first location then append a move command if not append a line command
             if (first) {
-                path = "M" + x + "," + y;
+                pathBuilder.append("M").append(x).append(SeparatorConstants.COMMA);
                 first = false;
             } else {
-                path += "L" + x + "," + y;
+                pathBuilder.append("L").append(x).append(SeparatorConstants.COMMA);
```

Comment:
I would also move the append for `x`, and comma to after the if-else (where the append for `y` is) as these are the same for both paths